vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/ReferenceMetadataDisplay.java

```diff
@@ -0,0 +1,11 @@
+package edu.asu.diging.vspace.referenceExpose;
+
+import edu.asu.diging.vspace.core.model.IReference;
+
+public class ReferenceMetadataDisplay {
+    
+    public static String urlEncodedRefMetaData(IReference ref) {
```

Comment:
why is this method static?

Reply:
I removed this class completely as I am using a registry class.